dev/com.ibm.ws.threading_policy_fat/test-applications/basicfat/src/web/PolicyExecutorServlet.java

```diff
@@ -2518,6 +2518,92 @@ public Void call() throws InterruptedException {
         assertTrue(future.isDone());
     }
 
+    // Tests behavior when shutdown occurs while timed invokeAll is submitting and running tasks. Supply a group of 5 tasks to invokeAll.
+    // The first task should complete successfully. The second task shuts down the executor. When the executor shuts down, the third and
+    // fourth tasks might or might not have been enqueued, depending on timing. However, the fifth task will not have been enqueued yet due to a
+    // maximum concurrency of 1 and maximum queue size of 2. The shutdown should prevent further enqueuing of tasks which causes the invokeAll
+    // operation to be rejected.
+    @SuppressWarnings("unchecked")
+    @Test
+    public void testShutdownDuringTimedInvokeAll() throws Exception {
+        PolicyExecutor executor = provider.create("testShutdownDuringTimedInvokeAll")
+                        .maxConcurrency(1)
+                        .maxQueueSize(2)
+                        .maxWaitForEnqueue(TimeUnit.SECONDS.toMillis(1))
+                        .queueFullAction(QueueFullAction.Abort);
+
+        AtomicInteger completedAfterShutdown = new AtomicInteger();
+        CountDownLatch unused = new CountDownLatch(0);
+        List<Callable<Object>> tasks = new ArrayList<Callable<Object>>();
+        tasks.add(Callable.class.cast(new SharedIncrementTask()));
+        tasks.add(Callable.class.cast(new ShutdownTask(executor, false, unused, unused, 0)));
+        tasks.add(Callable.class.cast(new SharedIncrementTask(completedAfterShutdown)));
+        tasks.add(Callable.class.cast(new SharedIncrementTask(completedAfterShutdown)));
+        tasks.add(Callable.class.cast(new SharedIncrementTask(completedAfterShutdown)));
+
+        try {
+            fail("shutdown should cause tasks submitted by invokeAll to be rejected. Instead: " +
+                 executor.invokeAll(tasks, 5, TimeUnit.MINUTES));
+        } catch (RejectedExecutionException x) {
+        }
+
+        assertEquals(1, SharedIncrementTask.class.cast(tasks.get(0)).count());
+
+        assertTrue(executor.awaitTermination(TIMEOUT_NS, TimeUnit.NANOSECONDS));
+
+        int count = completedAfterShutdown.get();
+        assertTrue(Integer.toString(count), count < 3);
+    }
+
+    // Tests behavior when shutdownNow occurs while invokeAll is submitting and running tasks. Supply a group of 3 tasks to invokeAll.
+    // The first task should complete successfully. The second task waits for the third to start and then shuts down the executor.
+    // Relying on behavior of policy executor that runs invokeAll tasks in reverse order order on the current thread if the global
```

Comment:
Extra "order" in this comment

Reply:
Thanks for catching the typo.  I'll get that corrected in the next set of changes since it's only in a comment and the build for this was green.